app/frontend/src/features/profile/profilePage/Home.tsx

```diff
@@ -0,0 +1,125 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+
+import Divider from "../../../components/Divider";
+import LabelAndText from "../../../components/LabelAndText";
+import { User } from "../../../pb/api_pb";
+import {
+  ABOUT_HOME,
+  ACCEPT_DRINKING,
+  ACCEPT_KIDS,
+  ACCEPT_PETS,
+  ACCEPT_SMOKING,
+  ADDITIONAL,
+  CAMPING,
+  HAS_HOUSEMATES,
+  HOST_DRINKING,
+  HOST_KIDS,
+  HOST_PETS,
+  HOST_SMOKING,
+  HOSTING_PREFERENCES,
+  HOUSE_RULES,
+  LAST_MINUTE,
+  LOCAL_AREA,
+  MAX_GUESTS,
+  PARKING,
+  SLEEPING_ARRANGEMENT,
+  SPACE,
+  TRANSPORTATION,
+  WHEELCHAIR,
+} from "../../constants";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  info: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-between",
+    width: "50%",
+  },
+}));
+
+interface HomeProps {
+  user: User.AsObject;
+}
+
+export default function Home({ user }: HomeProps) {
+  const classes = useStyles();
+
+  return (
+    <>
+      <div className={classes.root}>
+        <div className={classes.info}>
+          <Typography variant="h1">{HOSTING_PREFERENCES}</Typography>
+          <LabelAndText
+            label={LAST_MINUTE}
+            text={`${user.lastMinute?.value || ""}`}
+          />
+          <LabelAndText label={MAX_GUESTS} text={`${user.maxGuests}`} />
+          <LabelAndText
+            label={ACCEPT_SMOKING}
+            text={`${user.smokingAllowed}`}
+          />
+          <LabelAndText
+            label={ACCEPT_DRINKING}
+            text={`${user.drinkingAllowed}`}
```

Comment:
```suggestion
            text={`${user.drinkingAllowed?.value}`}
```
This is probably one of the culprit of that console error, since it's a `BoolValue.AsObject` rather than a primitive `boolean`, so you got to "unwrap" it. I'm surprised this error doesn't show up in dev 🤔